tests/test_resources/test_clusters/test_on_demand_cluster.py

```diff
@@ -233,3 +233,16 @@ def test_logs_surfacing_scheduler_basic_flow(self, cluster):
             "\n ", "\n"
         )
         assert cluster_logs_from_s3 in cluster_logs
+
+    @pytest.mark.level("minimal")
+    def test_change_autostop_of_saved_cluster(self, cluster):
+        cluster.save()
```

Comment:
if it's not an ondemand cluster we can just skip the test. also if its an ondeand cluster why woulnd't sky already be installed here?

Reply:
The test is a part of the on_demand_cluster test suite, so it will test only on-demand clusters anyway. 

Looks like when we set up the on-demand cluster, we don't sync / install skypilot to the cluster itself. (Moreover, running `cluster.run(["import sky"])` returns 1)